ocs_ci/helpers/dr_helpers.py

```diff
@@ -196,6 +227,14 @@ def relocate(
         drpc_obj = DRPC(
             namespace=namespace, resource_name=f"{workload_placement_name}-drpc"
         )
+    elif discovered_apps:
+        relocate_params = (
+            f'{{"spec":{{"action":"{constants.ACTION_RELOCATE}",'
+            f'"failoverCluster":"{old_primary}",'
+            f'"preferredCluster":"{preferred_cluster}"}}}}'
+        )
+        namespace = constants.DR_OPS_NAMESAPCE
+        drpc_obj = DRPC(namespace=namespace, resource_name=f"{workload_placement_name}")
```

Comment:
Relocation would be hanged in Relocating state unless the application is deleted, so how are we handling this here?

Reply:
we wait for STATUS_RELOCATING and then do cleanup